python/pyspark/sql/readwriter.py

```diff
@@ -1048,6 +1048,128 @@ def jdbc(self, url, table, mode=None, properties=None):
         self.mode(mode)._jwrite.jdbc(url, table, jprop)
 
 
+class DataFrameWriterV2(object):
```

Comment:
shall we move it to a new file?

Reply:
If you think that's better approach. I don't have strong opinion, though feature is small and unlikely to be used directly.